strapi/strapi
# Api token layout route

### What does it do?

- Adds a route to retrieve a Layout for the content api permissions

### How to test it?

when calling this endpoint `admin/content-api/permissions`
A structure similar to this one should be returned depending on the apis you have available:
```json
{
	"data": {
		"api::address": {
			"controllers": {
				"address": [
					"find",
					"findOne"
				]
			}
		},
		"api::category": {
			"controllers": {
				"category": [
					"find",
					"findOne",
					"create",
					"update",
					"delete",
					"createLocalization"
				]
			}
		},
	}
}
```
